deploy/helm/dynatrace-oneagent-operator/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: dynatrace-oneagent-operator
+description: The Dynatrace OneAgent Operator Helm chart for Kubernetes and Openshift
+type: application
+version: 0.1.0
+appVersion: 0.5.2
```

Comment:
Difference between `version` and `appVersion`?

Reply:
This version defines the version of the chart - the appversion defines the version of the app which is used